src/project-tests/tribute-page-tests.js

```diff
@@ -9,15 +9,15 @@ export default function createTributePageTests() {
     describe("#Tribute Page tests", function() {
 
         describe("#Content", function() {
-            it('1. My tribute page should have a main <div> element with corresponding id="main", which contains ' +
+            it('1. My tribute page should have a main element with corresponding id="main", which contains ' +
```

Comment:
@aaronRhodebeck also, let's change this to: `My tribute page should have an element...` so we're getting rid of the word `main` there, so we can't be accused later of a div working or another kind of element working when it really should be `<main>`. We don't really mean main in the context of the tag (even though it's sort of implied), so safer to get rid of it.